scss/_variables.scss

```diff
@@ -6,12 +6,18 @@ $grouparoo-yellow: #ff7518;
 $grouparoo-light-yellow: #fff8f1;
```

Comment:
I'm not sure these are compiling correctly.  The badge and button colors look different from Figma (they look like the Bootstrap defaults) and I wonder if this is the culprit.